pkgs/development/libraries/pangomm/default.nix

```diff
@@ -12,9 +12,8 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
-  nativeBuildInputs = [ pkg-config meson ninja python3 ] ++ lib.optional stdenv.isDarwin [
-    ApplicationServices
```

Comment:
Formatting-wise, I prefer always using optionals since they allow nicer chaining of multiple optionals. Also it makes less changelog noise when extra darwin services are added.

Reply:
makes sense to me, force pushed a commit

Reply:
This part should be reverted, as it regresses from the target of one input per line we are gradually switching to in GNOME packages.

Reply:
I'm not sure that I follow, is this what you wanted?

Reply:
Yeah, looks good now.